content/posts/type_safe_expressjs_genezio.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Type safe communication between multiple clients using ExpressJS and genezio
+date: 2024-04-02
+tags:
+  - Tutorials
+author: Bogdan Vlad 
+linkedIn: https://www.linkedin.com/in/iulian-bogdan-vlad
+/thumbnail: /images/web_scrape.webp
+preview: Maintaining an API across multiple clients can be a tedious task. Keeping the documentation in sync with the code, making sure that the data types passed in requests and response are the same in backend and frontend and handling errors in a consistent way are problems that any backend and frontend software engineer encountered at least once. What I want to present to you in this article, is a a zero config way using the genezio tool that can make things easier.
+# meta data start
+description: "Maintaining an API across multiple clients can be a tedious task. Keeping the documentation in sync with the code, making sure that the data types passed in requests and response are the same in backend and frontend and handling errors in a consistent way are problems that any backend and frontend software engineer encountered at least once. What I want to present to you in this article, is a a zero config way using the genezio tool that can make things easier."
```

Comment:
I am not exactly sure, but one of these should be shorter.

I remember vaguely that Paula told me that the description or preview should have 50-60 characters. We should check with her here.